userguide/payment/includes/payment-methods-flows.adoc

```diff
@@ -2,17 +2,18 @@ A *payment method* refers to the end-user's payment details. When an end-user en
 
 The https://github.com/killbill/killbill-api/blob/4ae1c343a593de937415e21feecb9f5405037fa3/src/main/java/org/killbill/billing/payment/api/PaymentApi.java#L541[PaymentApi#addPaymentMethod] is the entry point for saving a payment method in Kill Bill. Thus, a merchant website needs to invoke this method in order to save the payment details as a payment method in Kill Bill.
 
+[[payment_method_flows]]
 === Payment Method Flows Overview
 
 *Payment method flows* describe how a payment method gets created in Kill Bill.
 
 Kill Bill supports three main payment method flows as follows:
 
-* *Client-side tokenization* - In this case, the end user's payment method details are converted into a token which is sent to Kill Bill.
+* *Client-side Tokenization* - In this case, the end user's payment method details are converted into a token which is sent to Kill Bill.
```

Comment:
```suggestion
* *Client-Side Tokenization* – In this case, the end user's payment method details are converted into a token which is sent to Kill Bill.
```
I'm pasting in an "en dash" over the hyphen to see if you like it better. I like that it's a bit wider.